edit.php

```diff
@@ -70,26 +80,40 @@
                     <div class="row mb-3">
                         <label for="inputEmail3" class="col-sm-2 col-form-label">category</label>
                         <div class="col-sm-10">
-                            <input type="text" class="form-control" id="inputEmail3" name='category'>
+                            <input type="text" class="form-control" id="inputEmail3" name='category' value="<?php echo $result['category']?>">
                         </div>
                     </div>
                     <div class="row mb-3">
                         <label for="inputEmail3" class="col-sm-2 col-form-label">Image</label>
                         <div class="col-sm-10">
-                            <input type="text" class="form-control" id="inputEmail3" name='image'>
+                            <input type="text" class="form-control" id="inputEmail3" name='image' value="<?php echo $result['image']?>">
                         </div>
                     </div>
                     <div class="row mb-3">
                         <label for="inputEmail3" class="col-sm-2 col-form-label">description</label>
                         <div class="col-sm-10">
-
-                            <textarea type="text" name="description"></textarea>
+                           <textarea  name="description" cols="30" placeholder="Description">value="<?php echo $result['description']?>"</textarea>
```

Comment:
`value="<?php echo $result['description']?>"` should be `<?php echo $result['description']?>`